hudi-hadoop-common/src/test/java/org/apache/hudi/common/table/timeline/TestHoodieActiveTimeline.java

```diff
@@ -738,21 +745,18 @@ private List<HoodieInstant> getAllInstants() {
         // Following are not valid combinations of actions and state so we should
         // not be generating them.
         if (state == State.REQUESTED) {
-          if (action.equals(HoodieTimeline.SAVEPOINT_ACTION) || action.equals(HoodieTimeline.RESTORE_ACTION)
-              || action.equals(HoodieTimeline.ROLLBACK_ACTION)) {
+          if (action.equals(HoodieTimeline.SAVEPOINT_ACTION) || action.equals(HoodieTimeline.RESTORE_ACTION)) {
             continue;
           }
         }
-        if (state == State.INFLIGHT && action.equals(HoodieTimeline.ROLLBACK_ACTION)) {
-          continue;
-        }
```

Comment:
So previously the `ROLLBACK` is totally skipped.

Reply:
> So previously the `ROLLBACK` is totally skipped.

Yes, ROLLBACK was totally skipped before. I think it is unreasonable to ignore ROLLBACK, am I right?